http/http_parser.pony

```diff
@@ -29,6 +29,8 @@ type _PayloadState is
   | _ExpectError         // Not valid HTTP format
   )
 
+primitive Error
```

Comment:
How about calling it ParseError to be more explicit?

Reply:
✔️ done